std/experimental/logger/core.d

```diff
@@ -1763,7 +1763,7 @@ private @property Logger stdThreadLocalLogImpl() @trusted
 {
     import std.conv : emplace;
 
-    static ubyte[__traits(classInstanceSize, StdForwardLogger)] _buffer;
+    static /*align(StdForwardLogger.alignof)*/ void[__traits(classInstanceSize, StdForwardLogger)] _buffer;
```

Comment:
If setting alignment explicitly doesn't work, maybe the following would:
`size_t[1 + (__traits(classInstanceSize, StdForwardLogger) - 1) / size_t.sizeof] _buffer`